src/base/static/components/place-detail/index.js

```diff
@@ -31,7 +31,10 @@ import {
   custom_components as customComponents,
 } from "config";
 
-import { surveyConfigSelector } from "../../state/ducks/survey-config";
```

Comment:
A lot of these changes are just renaming "surveyConfig" to "commentsSurveyConfig"